aspnetcore/host-and-deploy/iis/troubleshoot.md

```diff
@@ -15,14 +15,22 @@ uid: host-and-deploy/iis/troubleshoot
 
 By [Luke Latham](https://github.com/guardrex)
 
-This article provides instructions on how to diagnose an ASP.NET Core app startup issue when hosting with [Internet Information Services (IIS)](/iis).
+This article provides instructions on how to diagnose an ASP.NET Core app startup issue when hosting with [Internet Information Services (IIS)](/iis). The information in this article applies to hosting in IIS on Windows Server and Windows Desktop.
+
+Additional troubleshooting topics:
+
+[Troubleshoot ASP.NET Core on Azure App Service](xref:host-and-deploy/azure-apps/troubleshoot)  
+Although App Service uses the [ASP.NET Core Module](xref:fundamentals/servers/aspnet-core-module) and IIS to host apps, see the dedicated topic for instructions specific to App Service.
+
+[Error handling](xref:fundamentals/error-handling)  
```

Comment:
This doc doesn't actually mention using the debugger.

Reply:
True because it has the strong xplat focus.

We need another link here. Let me see what our best IIS Express/debugging topic is ... prob something in the VS docs. I'll try to fix it on the next commit.

Reply:
orf ... I don't see a good topic to link. I can't link this one: https://docs.microsoft.com/en-us/visualstudio/debugger/how-to-enable-debugging-for-aspnet-applications. We need a topic like that over there for ASP.NET Core apps.

Anywho ... I don't think we should hold this work up over it. I've added a note to my Host & Deploy tracking issue to TAL at this near the end. I'll come back to it. For this topic, this might be the best we can do at the moment.